src/app/static/src/app/store/stepper/stepper.ts

```diff
@@ -50,10 +51,16 @@ export const initialStepperState = (): StepperState => {
     }
 };
 
+export const initialDataExplorationStepperState = (): StepperState => {
+    return {
+        steps: [],
```

Comment:
Could see this empty array being confusing in the future - maybe just a comment to explain why there are apparently no steps?